### ISSUE TITLE
Online replay of the most recent Tetra League match

### ISSUE THREAD

**Issue by notypey** [tetrio/issues#373]
### Is your feature request related to a problem? Please describe.
Not a problem technically, but it is kind of frustrating to not being able to review the last match played and work on improving base on that. 

### Describe the solution you'd like
Just the replay of the last played match in Tetra League in the style of 40l replay, which means with the Stats overview and allowing speed changing during the replay. Have it open for download instead of playing online would also be fine. 

### Describe alternatives you've considered
Basically anything that can show me the tetra league matches replay. Having obs on during every game is not quite an option since it would make the game too laggy, at least for someone like me with a crappy laptop.
### Additional context
IDK maybe keep it in the Tetra Channel / Me section

**Comment by o5k** [comment]
Dupe of #296
